AccountService.Domain/Services/ServiceAccount.cs

```diff
@@ -42,15 +41,17 @@ public async Task RegisterUser(RegisterApiModel userRequest)
                 if (user.UserData.ServiceNumber == null)
                 {
                     await _userManager.AddToRoleAsync(user, "participant");
+                    return new ResponseApiModel<HttpStatusCode>(HttpStatusCode.OK, true, "You have succesfully registered");
```

Comment:
You can get message for RestException from Resources.resx

Reply:
Done